magefiles/utils.go

```diff
@@ -150,3 +165,120 @@ func renderTemplate(destination, templatePath string, templateData interface{},
 
 	return nil
 }
+
+// helper function used for testing jvm-build-service PRs
+func overridePipelineBundle() error {
+	const buildTemplatesBundleKustomizeUrl = "https://raw.githubusercontent.com/redhat-appstudio/infra-deployments/main/components/build-templates/kustomization.yaml"
```

Comment:
nit: It may be worth comment with a summary of the overall flow this function is going to accomplish?